README.md

```diff
@@ -18,3 +19,63 @@ pip install -e .
 GITHUB_TOKEN=... review-gator --config branches.yaml
 ```
 
+Basic Configuration
+-------------------
+
+Configuration is provided via a YAML file. To see an example, run 
+`review-gator --config-skeleton`
+
+```
+launchpad:
+    owners:
+        # Launchpad username
+        LAUNCHPAD-USERNAME:
+            max-age: 30
+    branches:
+        # Full path to repo
+        LAUNCHPAD-BZR-REPO-PATH:
+            review-count: 2
```

Comment:
Please describe the function of:

* 'owners' vs 'branches
* max-age
* review-count

Reply:
@j5awry I would still like to see the field documented.